lightwood/encoder/time_series/__init__.py

```diff
@@ -1,5 +1,5 @@
 from lightwood.encoder.time_series.rnn import TimeSeriesEncoder
-from lightwood.encoder.time_series.plain import TimeSeriesPlainEncoder
+from lightwood.encoder.array.array import ArrayEncoder
 
 
-__all__ = ['TimeSeriesEncoder', 'TimeSeriesPlainEncoder']
+__all__ = ['TimeSeriesEncoder', 'ArrayEncoder']
```

Comment:
the fact array is within the time_series init is very weird given the goal of array is supposed to be sequences in general.

Reply:
Again, I think this is a mistake, it should indeed not be here